Extension/ui/settings.html

```diff
@@ -0,0 +1,131 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src vscode-resource:; script-src 'nonce-{{nonce}}';">
+  </head>
+
+  <!-- todo: create .css for styles -->
+  <style>
+    th, td {
+      padding: 10px;
+      font-family: verdana, sans-serif;
+      font-size: 15px;
+    }
+  </style>
+
+  <body>
+
+  <!-- todo: change table into div elements -->
+  <table>
+
+  <!-- row 0 -->
+  <!-- Header -->
+  <tr>
+      <td>
+        <table>
+          <td>
+            <img src="{{root}}/ui/LanguageCCPP_color_128x.png" alt="Microsoft C/C++ Extension" title="Microsoft C/C++ Extension">

```

Comment:
It looks like you added another copy of this image into the repo.  Can we keep just one?

Reply:
removed copy in ui folder